source/polling-srv-records-for-mongos-discovery/polling-srv-records-for-mongos-discovery.rst

```diff
@@ -78,10 +78,15 @@ discovery section of the original specification. The behaviour of the periodic
 rescan is similar, but not identical to the behaviour of initial seedlist
 discovery.  Periodic scan MUST follow these rules:
 
-- The driver will query the DNS server for SRV records on
+- If the initial topology was created through a ``mongodb+srv://`` URI,
+  the driver will query the DNS server for SRV records on
   ``{hostname}.{domainname}``, prefixed with ``_mongodb._tcp.``:
   ``_mongodb._tcp.{hostname}.{domainname}``.
 
+- If the initial topology was created through a ``mongodb+rawsrv://`` URI,
+  the driver will query the DNS server for SRV records using the original
+  URI without the scheme: ``_{servicename}._{protocol}.{hostname}.{domainname}``.
```

Comment:
I find this phrasing ("without the scheme" ) and the example confusing.  Maybe "using the original URI without any additional prefix" and no example would be clearer.  (If we get rid of `mongodb+srvraw`, then the example above should be changed to not hard-code _mongodb but to use the SRV service name.)